Mathlib/Geometry/Manifold/Instances/Sphere.lean

```diff
@@ -295,6 +297,14 @@ theorem stereographic_neg_apply (v : sphere (0 : E) 1) :
   ext1
   simp
 
+/-- The inverse stereographic projection is injective. -/
+theorem stereo_inj (hv : ‖v‖ = 1) :
+    Injective (stereoInvFun hv) :=
+  fun x y h =>
+  (stereographic hv).right_inv' (show x ∈ (stereographic hv).target by trivial) ▸
+  (stereographic hv).right_inv' (show y ∈ (stereographic hv).target by trivial) ▸
+  congrArg (stereographic hv).toPartialEquiv h
```

Comment:
I think we should regard `stereoInvFun` as an implementation detail en route to `stereographic` and so I think this should be spelled using `stereographic` since it has been constructed by now.

Furthermore I think there is a more basic lemma to abstract, namely:
```lean
lemma PartialEquiv.injective_symm_of_target_eq_univ
    {α β : Type*} (e : PartialEquiv α β) (h : e.target = Set.univ) :
    Injective e.symm := by
  simpa [Set.injective_iff_injOn_univ, h] using e.symm.injOn
```
Using that I think the proof here can be:
```suggestion
theorem injective_stereographic_symm (hv : ‖v‖ = 1) :
    Injective (stereographic hv).symm :=
  (stereographic hv).injective_symm_of_target_eq_univ rfl
```